src/common.h.in

```diff
@@ -238,6 +241,9 @@ typedef struct sr_mod_data_dep_s sr_mod_data_dep_t;
 /** static initializer of the shared memory structure */
 #define SR_SHM_INITIALIZER {.fd = -1, .size = 0, .addr = NULL}
 
+/** SR_CONN_ID_NONE is a reserved value representing an invalid or unset value. */
+#define SR_CONN_ID_NONE 0
+
```

Comment:
This is probably not needed at all, all the other IDs have 0 as invalid value without a specific macro. Or it can be a shared one for all of them.

Reply:
Ok.  I've removed the macro.